EleksTubeHAX/Mqtt_client_ips.cpp

```diff
@@ -160,7 +160,9 @@ void callback(char* topic, byte* payload, unsigned int length) {  //A new messag
     Serial.print("/");
     Serial.println(message);
 #endif    
-    
+  
+    if (tokensNumber < 3) return; // otherwise code below crashes on the strmp on non-initialized pointers in tokens[] array
```

Comment:
Formatting:
```suggestion
    if (tokensNumber < 3) {
        // otherwise code below crashes on the strmp on non-initialized pointers in tokens[] array
        return; 
    }
```